examples/agent/websocket/no_mic/main.go

```diff
@@ -182,8 +182,8 @@ func (dch MyHandler) Run() error {
 					0x10, 0x00, 0x00, 0x00, // Chunk size (16)
 					0x01, 0x00, // Audio format (1 for PCM)
 					0x01, 0x00, // Number of channels (1)
-					0x80, 0x5d, 0x00, 0x00, // Sample rate (24000)
```

Comment:
I fixed this as the output of this speech was way to fast, so this makes it sound non-chimpunk like.